types/defines/ai.d.ts

```diff
@@ -0,0 +1,1022 @@
+      "@cf/meta/llama-2-7b-chat-int8",
+      "@cf/mistral/mistral-7b-instruct-v0.1",
+      "@cf/mistral/mistral-7b-instruct-v0.1-vllm",
+      "@cf/mistral/mistral-7b-instruct-v0.2-lora",
+      "@cf/meta/llama-2-7b-chat-fp16",
+      "@hf/thebloke/llama-2-13b-chat-awq",
+      "@hf/thebloke/zephyr-7b-beta-awq",
+      "@hf/thebloke/mistral-7b-instruct-v0.1-awq",
+      "@hf/thebloke/codellama-7b-instruct-awq",
+      "@hf/thebloke/openchat_3.5-awq",
+      "@hf/thebloke/openhermes-2.5-mistral-7b-awq",
+      "@hf/thebloke/neural-chat-7b-v3-1-awq",
+      "@hf/thebloke/llamaguard-7b-awq",
+      "@hf/thebloke/deepseek-coder-6.7b-base-awq",
+      "@hf/thebloke/deepseek-coder-6.7b-instruct-awq",
+      "@hf/nousresearch/hermes-2-pro-mistral-7b",
+      "@hf/mistral/mistral-7b-instruct-v0.2",
+      "@cf/mistral/mixtral-8x7b-instruct-v0.1-awq",
+      "@hf/google/gemma-7b-it",
+      "@hf/nexusflow/starling-lm-7b-beta",
+      "@cf/deepseek-ai/deepseek-math-7b-instruct",
+      "@cf/defog/sqlcoder-7b-2",
+      "@cf/openchat/openchat-3.5-0106",
+      "@cf/tiiuae/falcon-7b-instruct",
+      "@cf/thebloke/discolm-german-7b-v1-awq",
+      "@cf/qwen/qwen1.5-0.5b-chat",
+      "@cf/qwen/qwen1.5-1.8b-chat",
+      "@cf/qwen/qwen1.5-7b-chat-awq",
+      "@cf/qwen/qwen1.5-14b-chat-awq",
+      "@cf/tinyllama/tinyllama-1.1b-chat-v1.0",
+      "@cf/microsoft/phi-2",
+      "@cf/google/gemma-2b-it-lora",
+      "@cf/google/gemma-7b-it-lora",
+      "@cf/meta-llama/llama-2-7b-chat-hf-lora",
+      "@cf/sven/test"
+    ];
+    readonly class: typeof AiTextGeneration;
+  };
+  readonly translation: {
+    readonly models: readonly ["@cf/meta/m2m100-1.2b"];
+    readonly class: typeof AiTranslation;
+  };
+  readonly summarization: {
+    readonly models: readonly ["@cf/facebook/bart-large-cnn"];
+    readonly class: typeof AiSummarization;
+  };
+  readonly "image-to-text": {
+    readonly models: readonly [
+      "@cf/unum/uform-gen2-qwen-500m",
+      "@cf/llava-hf/llava-1.5-7b-hf"
+    ];
+    readonly class: typeof AiImageToText;
+  };
+};
+/**
+ * modelAliases is where we define model aliases. This is useful when a model is renamed or we want to upgrade it to a
+ * newer version that is compatible with the older one.
+ * The key of the object is always the model that Workers AI uses internally that is defined in modelMappings.
+ */
+declare const modelAliases: {
+  "@hf/mistral/mistral-7b-instruct-v0.2": string[];
+};
+declare enum TensorType {
+  String = "str",
+  Bool = "bool",
+  Float16 = "float16",
+  Float32 = "float32",
+  Int16 = "int16",
+  Int32 = "int32",
+  Int64 = "int64",
+  Int8 = "int8",
+  Uint16 = "uint16",
+  Uint32 = "uint32",
+  Uint64 = "uint64",
+  Uint8 = "uint8",
+}
+declare type TensorOpts = {
+  shape?: number[];
+  name?: string;
+  validate?: boolean;
+};
+declare type TensorsObject = {
+  [name: string]: Tensor<any>;
+};
+declare const TypedArrayProto: any;
+declare function isArray(value: any): boolean;
```

Comment:
Irrelevant methods like this shouldn't be included

Reply:
Unused/irrelevant methods removed 👍